Exercise_A/index.html

```diff
@@ -16,7 +17,11 @@
   </head>
   <body>
     <!-- CODE HERE -->
```

Comment:
In this exercise we were meant to create two pages that contain different content.
You have done Ex B here

Please submit the correct exercises within the right folders.